src/modules/search/hooks/useDebounce.js

```diff
@@ -18,7 +18,7 @@ export function useDebounce(value, delay) {
     return () => {
       clearTimeout(timeoutHandler.current);
     };
-  }, [value, delay]);
+  }, [JSON.stringify(value), delay]);
```

Comment:
Why stringify the value?

Reply:
Yea, i dont like this either, doing this for all places using this hook while the only place this is needed is in the EditStake component?
Looks like we can handle this outside of the component?
Something like:
`const stakeValue = useDebounce(convertToBaseDenom(stakeAmount.value, token), 500);` 
instead of 
`const debouncedQueryConfig = useDebounce(queryConfig, 500);` 

Or maybe extend the useDebounce hook, so this could work
`const debouncedQueryConfig = useDebounce(queryConfig, 500, queryConfig.config.params.stake);`